examples/objectdetection/src/Photo.tsx

```diff
@@ -89,28 +90,78 @@ export const Photo: React.FC<Props> = () => {
       )}
       {screenState === "error" && (
         <>
-          <Text style={styles.errorText}>Error! Please try again.</Text>
-          <Pressable style={styles.selectButton} onPress={onClickSelectPhoto}>
-            <Text style={styles.selectButtonText}>Select a photo</Text>
-          </Pressable>
-        </>
+        <View style={styles.errorBox}>
+          <Text style={styles.errorText}>Error!</Text>
+          <Text style={styles.errorInfoText}>Please make sure you have selected a photo.</Text>
+        </View>
```

Comment:
This doesn't really seem right and also seems like it will conflict with #82 #98